src/AudioUnit/AudioComponent.cs

```diff
@@ -353,6 +369,12 @@ public AppKit.NSImage GetIcon ()
 		{
 			return new AppKit.NSImage (AudioComponentGetIcon (handle));
 		}
+
+		[Mac (11,0)]
+		public AppKit.NSImage CopyIcon () { 
+			var ptr = AudioComponentCopyIcon (handle);
+			return ptr == IntPtr.Zero ? null : new AppKit.NSImage (ptr);
+		}
```

Comment:
you can merge both method if you add `using NSImage = UIKit.UIImage;` in the `#if MONOMAC` on top of the file

Reply:
There is some extra ugliness, but sure, can be done.